### PR TITLE
[509630] Avoid NCDFE when o.e.jdt.debug is not present

### PR DESCRIPTION
**Pull request by kthoms**
Signed-off-by: Karsten Thoms <karsten.thoms@itemis.de>

### CODE REVIEW

**Comment by eclipsewebmaster** [comment]
Issue tracker reference:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=509630

File: org.eclipse.xtext.xbase.ui/src/org/eclipse/xtext/xbase/ui/debug/XbaseBreakpointPropertyTester.java [hunk 1]
```diff
@@ -18,6 +19,9 @@
 
 	@Override
 	public boolean test(Object receiver, String property, Object[] args, Object expectedValue) {
+		if (Platform.getBundle("org.eclipse.jdt.debug")==null) {
```
Comment (cdietrich) [inline org.eclipse.xtext.xbase.ui/src/org/eclipse/xtext/xbase/ui/debug/XbaseBreakpointPropertyTester.java]: Formatting

**Comment by cdietrich** [comment]
Besides the formatting LGTM

**Comment by kthoms** [comment]
Updated

**Pull request merged**